src/processors/data-processor.ts

```diff
@@ -58,20 +63,20 @@ const IGNORE_DATA_TYPES = Object.keys(AnalyzerDATA_TYPES).filter(
 
 export const PARSE_FIELD_VALUE_FROM_STRING = {
   [ALL_FIELD_TYPES.boolean]: {
-    valid: d => typeof d === 'boolean',
-    parse: d => d === 'true' || d === 'True' || d === 'TRUE' || d === '1'
+    valid: (d: any) => typeof d === 'boolean',
+    parse: (d: any) => d === 'true' || d === 'True' || d === 'TRUE' || d === '1'
   },
   [ALL_FIELD_TYPES.integer]: {
-    valid: d => parseInt(d, 10) === d,
-    parse: d => parseInt(d, 10)
+    valid: (d: any) => parseInt(d, 10) === d,
```

Comment:
same below, use `(d: unknown): boolean` for validate

Reply:
fixed